components/GitHubAccelerator.js

```diff
@@ -19,7 +19,7 @@ export default function GitHubAccelerator() {
         <div className="-mt-12 sm:-mt-16 sm:flex sm:items-end sm:space-x-5">
           <div className="flex">
             <Image
-              className="h-24 w-24 rounded-full ring-4 ring-white sm:h-32 sm:w-32"
+              className="h-24 w-24 rounded-full ring-4 ring-primary-low sm:h-32 sm:w-32"
```

Comment:
Good spot 👍 Do we need a `dark` version?

Reply:
Hi Eddie, these changes (and others) were made in this PR, but it seems it wasn't merged so the changes were not applied to the repo.

Consider viewing the related repo [here](https://github.com/EddieHubCommunity/LinkFree/pull/7791)

Reply:
Ok, please can you remove those changes that are not part of this PR

Reply:
Requested change has been implemented.